backend/siarnaq/api/compete/views.py

```diff
@@ -315,6 +316,59 @@ def scrimmage(self, request, pk=None, *, episode_id):
         serializer = self.get_serializer(page, many=True)
         return self.get_paginated_response(serializer.data)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="team_id",
+                type=int,
+                description="A team to filter for. Defaults to your own team.",
+            ),
+        ],
+        responses={
+            status.HTTP_204_NO_CONTENT: "No ranked matches found.",
+            status.HTTP_200_OK: HistoricalRatingSerializer(many=True),
+        },
+    )
+    @action(
+        detail=False,
+        methods=["get"],
+        permission_classes=(IsEpisodeMutable,),
+    )
+    def historical_ranking(self, request, pk=None, *, episode_id):
```

Comment:
ranking -> rating for consistency?

Reply:
++
in fact _please_ call it rating! in other places we use "ranking" for an ordinal ranking (eg 1st 2nd 3rd) so this could get really confusing to devs and users otherwise